frontend/src/tests/components/Reports/ReportLineTable.test.js

```diff
@@ -36,7 +38,7 @@ describe("ReportLineTable tests", () => {
 
     expect(screen.getByTestId(`${testId}-cell-row-0-col-userId`)).toHaveTextContent("1");
     expect(screen.getByTestId(`${testId}-cell-row-0-col-username`)).toHaveTextContent("Phill Conrad");
-    expect(screen.getByTestId(`${testId}-cell-row-0-col-totalWealth`)).toHaveTextContent("9745");
+    expect(screen.getByTestId(`${testId}-cell-row-0-col-totalWealth`)).toHaveTextContent("$9,745.00");
     expect(screen.getByTestId(`${testId}-cell-row-0-col-numOfCows`)).toHaveTextContent("3");
     expect(screen.getByTestId(`${testId}-cell-row-0-col-avgCowHealth`)).toHaveTextContent("100");
```

Comment:
are we okay with 100 health being an integer? would we want a decimal or percentage? (this might be out of scope of this ticket idk)

Reply:
Yeah, I think I'd leave that for now.